packaging/Dockerfile.fedora_29_git_rpm

```diff
@@ -0,0 +1,20 @@
+# This Dockerfile is based on the recommendations provided in the
+# Centos official repository (https://hub.docker.com/_/centos/).
+# It enables systemd to be operational.
+FROM fedora:29
+ENV container docker
+COPY docker_enable_systemd.sh docker_sys_config.sh ./
+RUN bash ./docker_enable_systemd.sh && rm ./docker_enable_systemd.sh -f
+
+# libibverbs is a dependency for openvswitch that is missing as a dependency in the RPM
+RUN yum -y upgrade && \
```

Comment:
Don't we need to use `dnf`?

Reply:
It is the same.

Reply:
Fedora abandon `dnf` in favour of `yum` as an alias? I remember we needed to install a special rpm to use `yum` on Fedora.